package.json

```diff
@@ -22,8 +22,9 @@
         "@aws-sdk/client-cloudwatch": "^3.388.0",
         "@aws-sdk/client-secrets-manager": "^3.195.0",
         "@aws-sdk/client-sqs": "^3.137.0",
+        "@balancer-labs/balancer-maths": "^0.0.6",
         "@balancer-labs/sdk": "github:beethovenxfi/balancer-sdk#beethovenx-master",
-        "@balancer/sdk": "^0.14.0",
+        "@balancer/sdk": "0.20.5",
```

Comment:
@franzns - can you please confirm this is not an issue for other parts of the code?

Reply:
the sdk is only used in the SOR part of the API